src/Components/input.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+
+const Input = () => (
+  <div className="wrapper-input">
+    <div className="result">
+      <h1>Result</h1>
+    </div>
```

Comment:
[ Optional Change ]
- [ ] What if you could have placed a `0` instead of the Result text? This area is expected to be holding numbers and not text characters. Kindly place a `0` there 👍🏽